assets/src/js/admin/reports/pages/overview-page/index.js

```diff
@@ -3,13 +3,76 @@
 // 12 column grid for content to exist in
 
 import Grid from '../../components/grid'
+import Card from '../../components/card'
+import Chart from '../../components/chart'
 
 const OverviewPage = () => {
     return (
         <Grid>
-            <div>
-                Child Component
-            </div>
+            <Card title="Doughnut Chart" width={4}>
```

Comment:
@henryholtgeerts can you internationalize this?

Reply:
@ravinderk Yes! Just pushed commits to internationalize Card titles and set translation for the reports app script.